scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -261,6 +261,43 @@ DFN-8-1EP_2x2mm_P0.5mm_EP0.6x1.2mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-8-1EP_2x2mm_P0.5mm_EP0.75x1.2mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.qorvo.com/products/d/da001904'
+  body_size_x:
+    nominal: 2
+    tolerance: 0.05
+  body_size_y:
+    nominal: 2
+    tolerance: 0.05
+  lead_width:
+    nominal: 0.25
+    tolerance: 0.05
+  lead_len:
+    nominal: 0.35
+    tolerance: 0.05
+
+  EP_size_x:
+    nominal: 0.75
```

Comment:
Please insert nominal package EP size (and tolerance) and add `EP_size_x_overwrite: 0.75` to get the suggested footprint pad.